src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier supplier = new ColorSupplier();
```

Comment:
Don't create several instances of the same class if you can use only one instance for your purposes. It's better to create a single instance of ColorSupplier outside of the getRandomBall method and reuse it. Consider making it a private class-level variable.